src/components/app.js

```diff
@@ -1,11 +1,17 @@
 ((() => {
   const html = `
     <div>
-      <h1>Hollaaa</h1>
+      <app-header></app-header>
+      <movies :image_url="image_url"></movies>
     </div>
   `
 
   Vue.component("tvinder-app", {
     template: html,
+    data(){
+      return {
+        image_url: "https://goo.gl/Puw6Ar"
```

Comment:
I also noticed that this isn't properly centering. I'm wondering if it is because the aspect ratio of the image has a larger horizontal value. We only dealt with vertical images in the OG Tvinder. 💃